tests/integration/README.md

```diff
@@ -0,0 +1,38 @@
+# Building the RPM package
+
+1. Checkout the project from OBS into some directory:
+```
+mkdir -p ~/obs/
+cd ~/obs/
+osc -A https://api.suse.de co Devel:SMT:SLE-12-SMT
+```
+2. Make the tarball in git working directory by running `make package`
+3. Update the data from OBS by the tarball and metadata from git:
+```
+cp package/* ~/obs/Devel:SMT:SLE-12-SMT/smt
+```
+4. Build the RPM package by running `osc build` in the OBS working directory.
+
+# Testing the RPM package
+
+1. The docker image expects the RPM package to be present in the following
+location:
+```
+smt_server/files/smt_current.rpm
+```
+2. Build docker images with `docker-compose build`
+3. Run the containers with `docker-compose up`
+4. After the containers finish loading, find out the name of the container
+by running `docker-exec ps`
+5. To get the shell in the running container run:
+```
+docker exec -it [container_name] /bin/bash
+```
+6. Inside the container run the test suite inside `smt_server`
+and `smt_client` containers, i.e.:
+```
+cd /rspec
+bundle
+rspec spec/smt_client # for the client container
+rspec spec/smt_server # for the server container
+```
```

Comment:
The manual is missing a hint to add values to `.env.example `. It would be good if the script exits when the values.